index.html

```diff
@@ -0,0 +1,1543 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/main.css">
+    <title>Twitter</title>
+</head>
+
+<body>
+    <div class="container ">
+        <header class="pd-sd-md">
+            <ul class="nav">
+                <li>
+                    <a href="https://twitter.com" target="_blank" rel="noopener noreferrer">
+                        <!-- Twitter Logo -->
+                        <svg viewBox="0 0 24 24" aria-hidden="true" class="icon twitter-color">
```

Comment:
Do you have any particular resources that you used for svg's?

Reply:
I used twitter's svg from their page